bin/bench.sh

```diff
@@ -300,16 +316,21 @@ run_reports() {
 
 cd $SCRIPT_DIR/..
 
-USE_GAUGE=0
 USE_GIT_CABAL=1
 USE_NIX=0
 set_common_vars
 
-DEFAULT_FIELDS="allocated bytescopied cputime"
-ALL_FIELDS="$FIELDS time cycles utime stime minflt majflt nvcsw nivcsw"
```

Comment:
Was this incorrect initially? `FIELDS` defined after `ALL_FIELDS`?

Reply:
yes.